test/e2e/framework.go

```diff
@@ -587,7 +587,7 @@ func (data *TestData) mutateFlowAggregatorConfigMap(ipfixCollector string) error
 	}
 	flowAggregatorConf, _ := configMap.Data[flowAggregatorConfName]
 	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#externalFlowCollectorAddr: \"\"", fmt.Sprintf("externalFlowCollectorAddr: \"%s\"", ipfixCollector), 1)
-	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#flowExportInterval: 60s", "flowExportInterval: 5s", 1)
+	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#flowExportInterval: 60s", "flowExportInterval: 4s", 1)
```

Comment:
could you use aggregatorExportInterval constant here?

Reply:
Sure, also change to use constant for exporterExportInterval.